ckanext/canada/schemas/presets.yaml

```diff
@@ -3457,6 +3457,8 @@ presets:
     - value: ODT
       mimetype: application/vnd.oasis.opendocument.text
       replaces: [odt]
+    - value: PBIX
+      replaces: [pbix]
```

Comment:
these `replaces` lists were only used for importing data from an earlier system

Reply:
@wardi we need the replaces for the Format guessing as the Core CKAN guess format does not return them in uppercase. And there are cases in which the guessing format could return jpeg instead of jpg, in which case we want the replacement.